modules/common-fs2/src/test/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/config/Base64HoconSpec.scala

```diff
@@ -26,10 +26,10 @@ class Base64HoconSpec extends Specification {
       Argument[Base64Hocon].read(input).toEither must beRight
     }
 
-    "parse a plain string as a HOCON source" in {
-      val inputStr = "+"
+    "not parse any other base-64 encoded string as a HOCON source" in {
+      val inputStr = "xyz"
       val input = getEncoder.encodeToString(inputStr.getBytes())
-      Argument[Base64Hocon].read(input).toEither must beRight
+      Argument[Base64Hocon].read(input).toEither must beLeft
     }
   }
 }
```

Comment:
Neat: add a test with a hocon file (in `test/resources`) that is to be resolved and check that it is [when the environment variable is set](https://stackoverflow.com/questions/37330416/set-default-env-variable-for-test-configuration-in-sbt).

Reply:
Added 2 tests.